addons/portal_rating/controllers/portal_chatter.py

```diff
@@ -44,3 +45,9 @@ def portal_message_fetch(self, res_model, res_id, domain=False, limit=False, off
         result = super(PortalChatter, self).portal_message_fetch(res_model, res_id, domain=domain, limit=limit, offset=offset, **kw)
         result.update(self._portal_rating_stats(res_model, res_id, **kw))
         return result
+
+    def _setup_portal_message_fetch_extra_domain(self, data):
+        domains = [super()._setup_portal_message_fetch_extra_domain(data)]
+        if data.get('rating_value', False) is not False:
```

Comment:
Is not this more future proof that way? I don't want to check if `rating_value` is truthful, I want to check if there is a set value, which is: any value besides False (as most things ORM-wise).

Reply:
> I don't see why it would be this explicit.

Because I don't want to check if the value is truthy but that it is set, which, in Odoo is the equivalent of "not False" (except for booleans which are always set I guess).

The fact that `rating_value` can't have the 0 value is a **current** implementation detail. The code can evolve and, from all we know, 0 could be used in custo already.